src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {
+  useState,
+  useCallback,
+  ChangeEventHandler,
+  ChangeEvent,
+} from 'react';
+import { useSelector } from 'react-redux';
+import { useAppDispatch, RootState } from '../../store/store';
```

Comment:
импорты, которые лежат далеко от это страницы, лучше импортировать по абсолютной ссылке
```suggestion
import { useAppDispatch, RootState } from 'src/store/store';
```

Reply:
Cannot find module 'src/store/store' or its corresponding type declarations.

Reply:
ну сделай правильно, чтобы нашелся